icepyx/tests/test_behind_NSIDC_API_login.py

```diff
@@ -51,7 +42,7 @@ def test_get_custom_options_output(session):
 # NOTE: best this test can do at the moment is a successful download with no errors...
 def test_download_granules_with_subsetting(reg, session):
     path = "./downloads_subset"
-    reg._session = session
+    reg.session = session
```

Comment:
do we still need to set this manually if login happens automatically now? Flagging this to double check.

Reply:
```suggestion
```